tools/release-ci/pkg/logging/gcs/finished.go

```diff
@@ -35,19 +36,21 @@ func UploadFinishedData(processLog, metadataFile, artifactDir string, passed boo
 
 	gcsPath := metadata.GcsPath()
 	uploadTargets := map[string]uploadFunc{
-		gcsPath + "/finished.json": dataUpload(finishedData),
+		ospath.Join(gcsPath, "finished.json"): dataUpload(finishedData),
```

Comment:
I hope to see the day we run this in a Windows container ;)

Reply:
it's not that :) ... actually path.Join will always use "/". The problem is we were getting duplicate "/"s